### DISCUSSION TITLE
Output for errors?

### DISCUSSION THREAD

**Discussion by Ilham-Habibullin** [zed-industries/zed#8797] [Feature Requests]
Is there a way to see zed's errors? Some developer mode that allows to see logs during running zed, if it's exist then how can i enable it?

**Comment by Moshyfawn** [comment]
`cmd-shift-p` to open the command palette:

- `zed: open log` to see the Zed logs.
- `debug: open language server logs` to open language server logs _(you can click on the server name at the top to switch between language server logs)_.
- `zed: open telemetry log` to open the telemetry logs.

**Reply by Ilham-Habibullin** [reply]
thanks!
